pkg/system/phase2_creating.go

```diff
@@ -853,6 +853,21 @@ func (r *Reconciler) ReconcileDBConfigMap(cm *corev1.ConfigMap, desiredFunc func
 func (r *Reconciler) SetDesiredPostgresDBConf() error {
 	dbConfigYaml := util.KubeObject(bundle.File_deploy_internal_configmap_postgres_db_yaml).(*corev1.ConfigMap)
 	r.PostgresDBConf.Data = dbConfigYaml.Data
+
+	overrideField := "noobaa-postgres.conf"
+	operator := &nbv1.NooBaa{}
+	if err := r.Client.Get(r.Ctx, types.NamespacedName{Namespace: options.Namespace, Name: "noobaa"}, operator); err != nil {
```

Comment:
the noobaa CR is already fetched at the beginning of the reconciler and you can access it through `r.NooBaa`. no need to get it again